lib/resources/dstu2/allergy_intolerance.yaml

```diff
@@ -0,0 +1,211 @@
+---
+field_name_base_url: http://hl7.org/fhir/DSTU2/allergyintolerance-definitions.html#AllergyIntolerance
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  example: AllergyIntolerance
+  note: resourceType must be AllergyIntolerance
+  url: http://hl7.org/fhir/DSTU2/json.html#resources
+
+- name: id
+  required:
+  - update: 'Yes'
+  - create: <b>Invalid</b>
+  cardinality: 0..1
+  type: id
+  description: The logical id of the resource to update.
+  example: '{"id": "123412"}'
+  note:
+  - update: The id value must match the AllergyIntolerance/&lt;id> value.
+  - create: The id field <b>must not</b> be set when performing an update operation.
```

Comment:
I know we talked about this a bit earlier.  After seeing this in the context of all the other fields, it seemed a lot more confusing to me than before when I thought we'd want to include it.  Maybe we want to convey this information in a note above/below the table instead of a row?